paasta_tools/cli/cmds/status.py

```diff
@@ -809,6 +817,47 @@ def _print_flink_status_from_job_manager(
         dashboard_url = metadata["annotations"].get("flink.yelp.com/dashboard_url")
         output.append(f"    URL: {dashboard_url}/")
 
+    cluster_without_pnw_dash = cluster.replace("pnw-", "")
+
+    # Print Flink config link resources
+    if verbose:
+        output.append(
+            f"    Yelpsoa configs: https://sourcegraph.yelpcorp.com/sysgit/yelpsoa-configs/-/tree/{service}"
+        )
+        output.append(
+            f"    Srv configs: https://sourcegraph.yelpcorp.com/sysgit/srv-configs/-/tree/ecosystem/"
```

Comment:
i'd probably link to GH here since it's easier to go back to the PR that edited a line from there

Reply:
i think we also never really type these as "srv configs" and "yelpsoa configs" on CI, but i guess whatever your users are expecting is fine 😔

Reply:
Updated to github links in
https://github.com/Yelp/paasta/pull/4063/commits/02c4363c0475992e91c646cba8beec658a330b34